index.md

```diff
@@ -16,98 +16,91 @@ order: 1
+Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Tools tend to offer different features and functionality which may help users to compare and assess web accessibility evaluation tools for their specific needs.
 
-The following features describe tools found on the [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/). 
-
-### Features You Can Use as a Filter
-
-You may use the following features to reduce the number of evaluation tools listed.
+The following describes tool features found on the [Web Accessibility Evaluation Tools List](https://www.w3.org/WAI/ER/tools/). 
 
 * **Guidelines**  
 Different organizations and governments may require conformance with different accessibility standards, thus different tools support these standards. The W3C Web Content Accessibility Guidelines (WCAG) is internationally recognized as the standard for web accessibility, and supported by most tools.
 
 * **Language**  
-Evaluation tools support different languages. This includes the user interface of the tools, as well as the language of the content they support (for example for detecting language-related barriers).
+The support for different languages provided by evaluation tools. This includes the user interface of the tools, as well as the language of the content they support (for example for detecting language-related barriers).
 
 * **Type of Tool**  
-Lists the platform and type of tools that are available, e.g. authoring tool or browser plugin, command line, desktop or mobile application.
+Evaluation tools can be plugins (extensions) for authoring tools and web browsers, command line tools, desktop or mobile applications, and online services. Diffent tools may be combined, depending on your design and development process.
 
-* **Supported Formats**  (referred to as "Technology" in the filters)
-Lists the format the tool can test, e.g. HTML, CSS, PDF, etc.
+* **Supported Formats**
+Most evaluation tools check the accessibility of HTML content. Some also support other web technologies, such as WAI-ARIA, CSS, SVG, and PDF.
 
 * **Assists by**  
-Lists what the specific uses of the tool, e.g. generating reports, providing step-by-step guidance, or displaying information within a web page
+Some web accessibility evaluation tools can be used in more than one way. The following are some common tool characteristics used to support the evaluation process:
 
-* **Automatically Checks**  
-The scope of what the evaluation tool can test. Sometimes a single page. Other times entire groups of related pages.
+** *Reports*. Reports generally list issues identified on many pages or whole sites. They are useful to help developers focus their attention on potential conformance issues can be automatically checked.
 
-* **License**  
```

Comment:
Reorganizing content - I fail to see why that was done and feel strongly it should have been left the way it was.

Reply:
Same as above.